src/index.ts

```diff
@@ -36,19 +38,32 @@ export async function scan(config: CheckstyleConfig) {
         await scanXmlReport(git, xmlReport, root, config.requireLineModification, (violation: Violation) => {
           var severity = violation.severity
```

Comment:
This variable is no longer used (and didn't serve a purpose before, as it seems).

Could you then remove this variable?

Reply:
I noticed that, but I was hesitant to remove it in case the code was doing something I didn't understand with it 😅 . It should be gone now.

Reply:
Alright, 👍 Thank you.